Изменения с целью создания автодокументации (HTML) и видео для шагов в цикле

StS-testim opened the pull request:
Версия без изменений не работает должным образом для автодокументирования с шагами в Цикле
реализация  #
Внесены правки в обработки : 

-  Автоинструкции.epf  (правки из-за изменений в vanessa-automation.epf)
-  РаботаСВидео.epf 
-  vanessa-automation.epf (правки в модуле Управляемая форма)
Все изменения с комментариями ЦВ_ПР.

StS-testim committed:
//+++ ЦВ_ПР 10.07.2025 Реализовано иначе, т.е. в общем стиле, См. Скриншот= в Процедура ДобавитьВЛогИнструкцииHTMLВыполненияШага (Управляемая форма)

StS-testim committed:
Исправления с цооменатриями ЦВ_ПР для создания автоинструкции с шагами в цикле

StS-testim committed:
Исправления с комментариями ЦВ_ПР для создания автоинструкций (HTML) и видеоинструкций с шагами в цикле.

Pr-Mex commented:
@StS-testim 
Реквест пытается удалить код, к которому он не относится
<img width="2553" height="1085" alt="image" src="https://github.com/user-attachments/assets/e6d8846a-f6c4-4d87-9820-c161418a5d20" />

StS-testim committed:
Уточнения формата добления лидирующих нулей для сценария и номера шага вместо 3 (4) сделали 5

StS-testim committed:
Добавлены изменения ЦВ_ПР в текущую версию формы

StS-testim committed:
Update Module.bsl

Изменения ЦВ_ПР к текущей версии формы "Управляемая форма"

StS-testim commented:
@Pr-Mex , 
Внесены правки к текущей версии (было к версии 42.19)

Pr-Mex commented:
run tests

Pr-Mex commented:
run tests

StS-testim committed:
Замена СтрНайти() на _СтрНайти() для режима совместимости 8.2

StS-testim committed:
Update Module.bsl

Замена СтрНайти() на _СтрНайти() для совместимости 8.2

Pr-Mex commented:
run tests

StS-testim committed:
Добавление нулей в имени файла scen_ до 5 знаков вместо 3(4)

StS-testim committed:
Восстановление версии Автоинструкции с уточнениями для HTML
Реализовано иначе, т.е. в общем стиле, См. также в Процедура ДобавитьВЛогИнструкцииHTMLВыполненияШага
(УправляемаяФорма)
Добавление лидирующих нулей в имени файла (5 вместо 3) Управляемая форма)

StS-testim committed:
Повторная отправка

StS-testim committed:
Повторная отправка Автоинструкции

Pr-Mex commented:
run tests

Pr-Mex commented:
@StS-testim 
Спасибо!
